### PR TITLE
#27 Added youtube embedded videos from pusher dotnetsheff playlist

### PR DESCRIPTION
**Pull request by sakisk**
Closes dotnetsheff/dotnetsheff#27

### CODE REVIEW

**Comment by sakisk** [comment]
@kevbite What happens if a video has a youtube as well as a wistia published version?

**Comment by sakisk** [comment]
@kevbite There's an animation when you scroll to next wistia videos which youtube ones don't have. Might need to add it to youtube css class or remove it from wistia ones?

**Comment by sakisk** [comment]
I've tried adding a [youtube plugin](https://gist.github.com/joelverhagen/1805814) which, i think, is a more elegant way to embed youtube videos from using youtube `iframe` straight up. For some reason couldn't pass the `youtube-id` value from videos collection when iterating to build the page. The tag worked when hardcoding the value...

**Comment by kevbite** [comment]
@sakisk they'll never be both a youtube and a wistia for one meetup, it's one or the other.

File: _data/videos.yml [hunk 1]
```diff
@@ -63,3 +63,123 @@
   date: 2017-09-05
   wistia-id: ubwutclg1j
   meetup-event-id: 238948950
+
```
Comment (kevbite) [inline _data/videos.yml]: Thanks for filling in the yml file! <3

**Comment by kevbite** [comment]
Could do with lazy loading the videos as all the iframes try to download the world. I'll, however, add another ticket for this as this at least shows the latest videos now! Good work! 👍

**Pull request merged**